addons/mail/static/src/models/channel_call_participant.js

```diff
@@ -0,0 +1,63 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, many, one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'ChannelCallParticipant',
+    identifyingFields: ['callParticipant', 'channel'],
+    recordMethods: {
+        async cancelInvitation() {
+            const channelData = await this.messaging.rpc(({
+                route: '/mail/rtc/channel/cancel_call_invitation',
+                params: {
+                    channel_id: this.channel.id,
+                    partner_ids: this.callParticipant.invitedPersona.partner && [this.callParticipant.invitedPersona.partner.id],
+                    guest_ids: this.callParticipant.invitedPersona.guest && [this.callParticipant.invitedPersona.guest.id],
+                },
+            }));
+            if (!this.channel.exists()) {
+                return;
+            }
+            this.channel.update(channelData);
```

Comment:
`this` might get destroyed then accessing channel will fail, `channel` should be saved before the rpc